python-sdk/pyproject.toml

```diff
@@ -48,7 +48,8 @@ tests = [
     "pytest-cov",
     "pytest-describe",
     "mypy",
-    "sqlalchemy-stubs"  # Change when sqlalchemy is upgraded https://docs.sqlalchemy.org/en/14/orm/extensions/mypy.html
+    "sqlalchemy-stubs",  # Change when sqlalchemy is upgraded https://docs.sqlalchemy.org/en/14/orm/extensions/mypy.html
+    "openlineage-airflow>=0.12.0"
```

Comment:
I don't think there's anything preventing you from using newest `0.15.1` 😉

Reply:
our repo supports `apache-airflow>=2.0` so I think its good to keep it